firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -237,21 +237,18 @@ void showFeedbackNotification(
    *   <li>Starts a full screen activity for the tester to compose and submit the feedback
    * </ol>
    *
-   * @param infoText text that will be shown to the tester before they submit feedback. If you’re a
+   * @param additionalFormText text that will be shown to the tester before they submit feedback. If you’re a
    *     customer who would like to provide notice to your testers about collection and processing
    *     of their feedback data, you can use this text to provide such notice.
    * @param interruptionLevel the level of interruption for the feedback notification. On platforms
    *     below Android 8, this corresponds to a notification channel importance and once set cannot
    *     be changed except by the user.
    */
   void showFeedbackNotification(
-      @NonNull CharSequence infoText, @NonNull InterruptionLevel interruptionLevel);
+      @NonNull CharSequence additionalFormText, @NonNull InterruptionLevel interruptionLevel);
 
   /**
    * Hides the notification shown with {@link #showFeedbackNotification}.
-   *
-   * <p>This should be called in the {@link Activity#onDestroy} of the activity that showed the
-   * notification.
```

Comment:
Did you mean to remove this?

Reply:
Good catch, I forgot to mention this in the PR description. Yes, I meant to remove this. We don't recommend calling this in `onDestroy()` anymore because it's not necessary now that we automatically hide the notification.